templates/registration/register.html

```diff
@@ -8,25 +8,37 @@ <h2 class="extra-padding">Register</h2>
             <hr>
         </div>
         <div class="register-buttons">
+            <p class="register-copy">
+                Sign up with one click using Facebook
+            </p>
             <div class="button-container">
+                <a class="facebook-font" href="{% url 'social:begin' backend='facebook-register' %}">
                 <div class="sign-up-button facebook">
-                    <h5 class="button-font"><a class="facebook-font" href="{% url 'social:begin' backend='facebook-register' %}"><i class="fa fa-facebook button-icon-padding" aria-hidden="true"></i>Sign Up with Facebook</a></h5>
```

Comment:
Ahhh, okay I missed that. You'll want to indent the divs that are now inside of the link.
```html
<a class="facebook-font" href="{% url 'social:begin' backend='facebook-register' %}">
    <div class="sign-up-button facebook">
        <h5 class="button-font"><i class=...></i>Sign Up with Facebook</h5>
    </div>
</a>
```

And do the same for the Google and email buttons.